osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -502,16 +492,12 @@ private BeatmapSetInfo importToStorage(ArchiveReader reader)
         /// <summary>
         /// Returns a list of all usable <see cref="BeatmapSetInfo"/>s.
         /// </summary>
-        /// <param name="populate">Whether returned objects should be pre-populated with all data.</param>
         /// <returns>A list of available <see cref="BeatmapSetInfo"/>.</returns>
-        public List<BeatmapSetInfo> GetAllUsableBeatmapSets(bool populate = true)
+        public List<BeatmapSetInfo> GetAllUsableBeatmapSets()

```

Comment:
why removing functionality?

Reply:
Eager loading is implemented in Query methods(https://github.com/UselessToucan/osu/blob/EfCore/osu.Game/Beatmaps/BeatmapStore.cs#L138).
If this is critical for you, I will try to extract it to Populate method.